springapp/src/test/java/com/javaacademy/robot/service/BookServiceTest.java

```diff
@@ -256,4 +256,40 @@ private BookModels getBookModels() {
         bookDto3.setIndustryIdentifier(3L);
         return new ArrayList<>(Arrays.asList(bookDto, bookDto2, bookDto3));
     }
+
+    @Test
+    public void sortAuthorsIfAppliedTest() {
+        BookService givenBookService = new BookService(bookRepository, bookConverter);
+        FilterOrder filterOrder = FilterOrder.ASCENDING;
+        String columnName = "authors";
+        List<String> strings = new ArrayList<>();
+        String first = "abc";
+        String second = "cba";
+        String third = "wtf";
```

Comment:
Not professional :rage3:

Reply:
What do you mean? Those are three, perfectly random, short, easy-to-distinct strings that will perfectly fit the case.
:suspect: